### ISSUE TITLE
undeclared identifier 'SIZE_MAX' on FreeBSD with 0.23.22

### ISSUE THREAD

**Issue by dmilith** [p11-glue/p11-kit#343]
```
p11-kit/lists.c:67:7: error: use of undeclared identifier 'SIZE_MAX'
        if ((SIZE_MAX - 1) / 3 < n_data)
```

**Comment by dmilith** [comment]
Fix:

Add ```
-DSIZE_MAX=__SSIZE_MAX
```
to CFLAGS

**Comment by ueno** [comment]
That should be fixed with #342. Thanks!
